admin_manual/groupware/circles/app_overview.rst

```diff
@@ -0,0 +1,152 @@
+========
+Overview
+========
+
+Circles is a Nextcloud App that allow Nextcloud users to generate their own grouping of users and contacts.
+
+
+
+Types of Circles
+^^^^^^^^^^^^^^^^
+
+.. note::
+ Once the Circles app is enabled, your users can creates 2 types of Circles: `Circle` and `Personal Circle`.
+ There is no way to switch from `Circle` to `Personal Circle` and vice versa.
+ `Personal Circle` are really specific and will be treated separately.
+
+
+When creating a Circle, its default configuration makes it 'Closed' and `Hidden`, meaning than no one
+can see it or join it but the Circle can already be used as an entity to share files and content.
+The user that created the Circle is the first and only Member and will have Owner rights that allows him/her to:
+
+- add/remove Members,
+- edit their Level,
+- change the settings of the Circle,
+- change its configuration
+
+.. _app_overview_type_circles:
+
+The configuration is done by enabling/disabling some flag that define the way other people view and join the Circle:
+
+ - **Visible** (8): Visible to everyone, if not visible, people have to know its name to be able to find
+   it.
+
+ - **Open** (16): Circle is open, people can join it.
+
+ - **Invite** (32): Adding a Member generate an invitation that needs to be accepted.
+
+ - **Request** (64): A request to join the Circle needs to be validated by a Moderator.
+
+ - **Friend** (128): Members of the Circle can invite their friends.
```

Comment:
"Friend" is a strong word here, not really suitable for enterprise environments.
The string might likely have to be adjusted in the app itself as well (if you agree)